### PR TITLE
Experimental focus states

### PR DESCRIPTION
**Pull request by lc512k**
@keirog Experimenting with Patricia's new focus styles. We'd also need an inverse style and a style for menu drawer items that are already `teal-1`. Maybe others too 🤔 

Current
current_focus

New
new_focus

Done the `gov.uk` way with background and outline
screen shot 2017-01-29 at 14 24 49

### CODE REVIEW

File: client/accessibility.scss [hunk 1]
```diff
@@ -26,21 +24,33 @@ $o-typography-is-silent: false;
...
-		-webkit-margin-start: auto;
-		-webkit-margin-end: auto;
-		border-style: inset;
-		border-width: 1px;
+
+	// EXPERIMENTAL FOCUS STYLES, TO BE IMPLEMENTED GLOBALLY
+	a {
+		&:focus,
+		&:active {
+			background-color: oColorsGetPaletteColor('teal-1');
```
Comment (keirog) [inline client/accessibility.scss]: Just as a note for the global implementation, these declarations would probably benefit from `!important`s, to lower the chance of specificity issues.

Comment (keirog) [inline client/accessibility.scss]: (Disclaimer: Not that I generally condone `!important`)

**Comment by keirog** [comment]
If only we could move away from using `teal-1` for arbitrary background styling. If we did, we could potentially avoid having an inverse focus style, and just use the teal background for every focussed link. I’ll ask the design folks what they think, as this is an inherited problem from yesteryear.

**Comment by keirog** [comment]
Scrap that: we’d still need an inverse version for these teasers:
inversed-teaser
😢

File: client/accessibility.scss (hunk 1 above)
Comment (lc512k) [inline client/accessibility.scss]: Good point @keirog . This stuff is probably one of those few things that actually do merit `!important`, isn't it?

**Comment by lc512k** [comment]
I've only just read the text in that blue teaser in the screenshot to realize it's 😢 for way more than just CSS reasons

**Pull request merged**